platform/ios/scripts/publish-cn.sh

```diff
@@ -0,0 +1,53 @@
+
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+#
+# make the framework
+#
+  
+step "Make the framework"
+export BUILDTYPE=Release
+./platform/ios/scripts/package.sh
+echo "making package"
+wait
+echo "finished!"
+
+#
+# zip
+#
+cd build/ios/pkg
+ZIP="mapbox-ios-sdk-cn.zip"
+step "Compressing ${ZIP}…"
+rm -f ../${ZIP}
+zip -yr ../${ZIP} *
+
+#
+# report file sizes
+#
+step "Echoing file sizes…"
+du -sh ../${ZIP}
+du -sch *
+du -sch dynamic/*
+
+#
+# upload
+#
+PROGRESS=
+if [ -n "${CI:-}" ]; then
+    PROGRESS="--no-progress"
+fi
+
+step "Uploading ${ZIP} to s3…"
+
+# Since this build is primarily for .CN customers, it will be hosted on .cn. 
+
+aws s3 cp ../${ZIP} s3://binary.mapbox.cn/ios/builds/maps-sdk/ --acl public-read ${PROGRESS}
+echo "URL:  https://s3.cn-north-1.amazonaws.com.cn/binary.mapbox.cn/${ZIP}"
```

Comment:
Given `s3://binary.mapbox.cn/ios/builds/maps-sdk/`, this isn’t the URL I would expect — is it accurate?

Reply:
@friedbunny see the discussion https://github.com/mapbox/mapbox-plugin-ios-china/issues/10#issuecomment-414992708

Reply:
Still, L52 appears to be uploading to `https://s3.cn-north-1.amazonaws.com.cn/binary.mapbox.cn/ios/builds/maps-sdk/mapbox-ios-sdk-4.5.0-cn.1.zip`, while L53 would print `https://s3.cn-north-1.amazonaws.com.cn/binary.mapbox.cn/mapbox-ios-sdk-4.5.0-cn.1.zip` (i.e., without the subfolders).